chain/client/src/client_actor.rs

```diff
@@ -20,6 +20,8 @@ use near_chain::chain::{
     BlockCatchUpResponse, StateSplitRequest, StateSplitResponse,
 };
 use near_chain::test_utils::format_hash;
+#[cfg(feature = "sandbox")]
```

Comment:
I think it may be a good idea to avoid using conditional imports.
I know, that we still use conditional imports here, but in general we are trying to phase it out.

Reply:
Sounds good, I moved all the other conditional imports inplaced into their respective areas